### DISCUSSION TITLE
Discussion for 0.11.0 Release

### DISCUSSION THREAD

**Discussion by susangreve** [go-vela/community#414] [General]
Space for discussion about the upcoming 0.11.0 release. 
Issues can be found [here](https://github.com/go-vela/community/issues)
Upvote the issues you would like to see in an upcoming release!

**Comment by susangreve** [comment]
- Better tracing between worker/ui/cli (Cargill)
- Consolidation of repos (see [here](https://github.com/go-vela/community/blob/master/proposals/2021/08-25_repo-structure.md))

**Comment by cognifloyd** [comment]
Extend the Runtime Engine interface to allow the kubernetes runtime to handle build-level (vs container-level) setup and assemble steps. See:
- https://github.com/go-vela/pkg-runtime/pull/148
- https://github.com/go-vela/pkg-executor/pull/179
